web/src/drafts.js

```diff
@@ -294,6 +293,7 @@ export function restore_draft(draft_id) {
     }
 
     const compose_args = restore_message(draft);
+    compose_args.draft_id = draft_id;
```

Comment:
I added the `draft_id` to the `compose_args` because it enables the call of the 
```
$("#compose-textarea").data("draft-id", opts.draft_id);
compose_validate.check_overflow_text();
```
within the `compose_actions.start()` function only when `opts.draft_id` returns true.

Reply:
Cool, but I think we actually want `restore_message` to do this; changed.